### PR TITLE
docs: Updated Ingress and Istio manifests to support Openshift 3.11

### PR DESCRIPTION
**Pull request by odubajDT**
Signed-off-by: odubajDT <ondrej.dubaj@dynatrace.com>

Fixes https://github.com/keptn/keptn/issues/4407

### CODE REVIEW

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Maybe it is time to deprecate documentation for old versions. Subject for a rework later. This change LGTM, but 0.x.12 docs also need to be updated after https://github.com/keptn/keptn.github.io/pull/1009

**Comment by odubajDT** [comment]
> Maybe it is time to deprecate documentation for old versions. Subject for a rework later. This change LGTM, but 0.x.12 docs also need to be updated after #1009

Sure, we can keep this open and after merging #1009, we will adapt the changes

**Comment by thisthat** [comment]
#1009 is merged, @odubajDT  you can rebase against master and edit the docs for 0.12.x 
When we release 0.12.x, we will remove 0.8.7 from the list of versions.

**Comment by odubajDT** [comment]
> #1009 is merged, @odubajDT you can rebase against master and edit the docs for 0.12.x When we release 0.12.x, we will remove 0.8.7 from the list of documented versions.

Updated

File: content/docs/0.10.x/operate/install/index.md [hunk 1]
```diff
@@ -403,6 +403,44 @@ Depending on whether you would like to install the execution plane for continuou
     </p>
     </details>
 
+    <details><summary>**Istio 1.8+ on OpenShift 3.11**</summary>
+    <p>
+
+    * Istio provides an Ingres Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
```
Comment (mowies) [inline content/docs/0.10.x/operate/install/index.md:409]:
```suggestion
    * Istio provides an Ingress Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
```

File: content/docs/0.10.x/operate/install/index.md [hunk 2]
```diff
@@ -403,6 +403,44 @@ Depending on whether you would like to install the execution plane for continuou
...
+
+    * [Determine the ingress IP](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
+
+      ```console
+    kubectl -n istio-system get svc istio-ingressgateway
+      ```
+
+    * Create an `ingress-manifest.yaml` manifest for an Ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example of an `ingress-manifest.yaml` manifest shown below, `nip.io` is used as wildcard DNS for the IP address.)
```
Comment (mowies) [inline content/docs/0.10.x/operate/install/index.md:417]:
```suggestion
    * Create an `ingress-manifest.yaml` manifest for an Ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example below, `nip.io` is used as wildcard DNS for the IP address.)
```

File: content/docs/0.10.x/operate/install/index.md [hunk 3]
```diff
@@ -419,6 +457,48 @@ Depending on whether you would like to install the execution plane for continuou
...
+
+    * To install an NGINX Ingress Controller, please refer to the [official documentation](https://kubernetes.github.io/ingress-nginx/).
+
+    * [Determine the ingress IP](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
+
+      ```console
+    kubectl -n ingress-nginx get svc ingress-nginx
+      ```
+
+    * Create an `ingress-manifest.yaml` manifest for an ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example of an `ingress-manifest.yaml` manifest shown next, `nip.io` is used as wildcard DNS for the IP address.)
```
Comment (mowies) [inline content/docs/0.10.x/operate/install/index.md:497]:
```suggestion
    * Create an `ingress-manifest.yaml` manifest for an ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example below, `nip.io` is used as wildcard DNS for the IP address.)
```

File: content/docs/0.9.x/operate/install/index.md [hunk 4]
```diff
@@ -419,6 +457,48 @@ Depending on whether you would like to install the execution plane for continuou
...
+
+    * To install an NGINX Ingress Controller, please refer to the [official documentation](https://kubernetes.github.io/ingress-nginx/).
+
+    * [Determine the ingress IP](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
+
+      ```console
+    kubectl -n ingress-nginx get svc ingress-nginx
+      ```
+
+    * Create an `ingress-manifest.yaml` manifest for an ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example of an `ingress-manifest.yaml` manifest shown next, `nip.io` is used as wildcard DNS for the IP address.)
```
Comment (mowies) [inline content/docs/0.9.x/operate/install/index.md:497]:
```suggestion
    * Create an `ingress-manifest.yaml` manifest for an ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example below, `nip.io` is used as wildcard DNS for the IP address.)
```

File: content/docs/0.9.x/operate/install/index.md [hunk 5]
```diff
@@ -403,6 +403,44 @@ Depending on whether you would like to install the execution plane for continuou
...
+
+    * Istio provides an Ingres Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
+
+    * [Determine the ingress IP](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
+
+      ```console
+    kubectl -n istio-system get svc istio-ingressgateway
+      ```
+
+    * Create an `ingress-manifest.yaml` manifest for an Ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example of an `ingress-manifest.yaml` manifest shown below, `nip.io` is used as wildcard DNS for the IP address.)
```
Comment (mowies) [inline content/docs/0.9.x/operate/install/index.md:417]:
```suggestion
    * Create an `ingress-manifest.yaml` manifest for an Ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example below, `nip.io` is used as wildcard DNS for the IP address.)
```

File: content/docs/0.8.x/operate/install/index.md [hunk 6]
```diff
@@ -419,6 +457,48 @@ Depending on whether you would like to install the execution plane for continuou
...
+
+    * To install an NGINX Ingress Controller, please refer to the [official documentation](https://kubernetes.github.io/ingress-nginx/).
+
+    * [Determine the ingress IP](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
+
+      ```console
+    kubectl -n ingress-nginx get svc ingress-nginx
+      ```
+
+    * Create an `ingress-manifest.yaml` manifest for an ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example of an `ingress-manifest.yaml` manifest shown next, `nip.io` is used as wildcard DNS for the IP address.)
```
Comment (mowies) [inline content/docs/0.8.x/operate/install/index.md:497]:
```suggestion
    * Create an `ingress-manifest.yaml` manifest for an ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example below, `nip.io` is used as wildcard DNS for the IP address.)
```

File: content/docs/0.8.x/operate/install/index.md [hunk 7]
```diff
@@ -403,6 +403,44 @@ Depending on whether you would like to install the execution plane for continuou
...
+
+    * Istio provides an Ingres Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
+
+    * [Determine the ingress IP](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
+
+      ```console
+    kubectl -n istio-system get svc istio-ingressgateway
+      ```
+
+    * Create an `ingress-manifest.yaml` manifest for an Ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example of an `ingress-manifest.yaml` manifest shown below, `nip.io` is used as wildcard DNS for the IP address.)
```
Comment (mowies) [inline content/docs/0.8.x/operate/install/index.md:417]:
```suggestion
    * Create an `ingress-manifest.yaml` manifest for an Ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example below, `nip.io` is used as wildcard DNS for the IP address.)
```

File: content/docs/0.8.x/operate/install/index.md [hunk 8]
```diff
@@ -403,6 +403,44 @@ Depending on whether you would like to install the execution plane for continuou
     </p>
     </details>
 
+    <details><summary>**Istio 1.8+ on OpenShift 3.11**</summary>
+    <p>
+
+    * Istio provides an Ingres Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
```
Comment (mowies) [inline content/docs/0.8.x/operate/install/index.md:409]:
```suggestion
    * Istio provides an Ingress Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
```

File: content/docs/0.12.x/operate/install/index.md [hunk 9]
```diff
@@ -419,6 +457,48 @@ Depending on whether you would like to install the execution plane for continuou
...
+
+    * To install an NGINX Ingress Controller, please refer to the [official documentation](https://kubernetes.github.io/ingress-nginx/).
+
+    * [Determine the ingress IP](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
+
+      ```console
+    kubectl -n ingress-nginx get svc ingress-nginx
+      ```
+
+    * Create an `ingress-manifest.yaml` manifest for an ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example of an `ingress-manifest.yaml` manifest shown next, `nip.io` is used as wildcard DNS for the IP address.)
```
Comment (mowies) [inline content/docs/0.12.x/operate/install/index.md:497]:
```suggestion
    * Create an `ingress-manifest.yaml` manifest for an ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example below, `nip.io` is used as wildcard DNS for the IP address.)
```

File: content/docs/0.12.x/operate/install/index.md [hunk 10]
```diff
@@ -403,6 +403,44 @@ Depending on whether you would like to install the execution plane for continuou
...
+
+    * Istio provides an Ingres Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
+
+    * [Determine the ingress IP](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
+
+      ```console
+    kubectl -n istio-system get svc istio-ingressgateway
+      ```
+
+    * Create an `ingress-manifest.yaml` manifest for an Ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example of an `ingress-manifest.yaml` manifest shown below, `nip.io` is used as wildcard DNS for the IP address.)
```
Comment (mowies) [inline content/docs/0.12.x/operate/install/index.md:417]:
```suggestion
    * Create an `ingress-manifest.yaml` manifest for an Ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example below, `nip.io` is used as wildcard DNS for the IP address.)
```

File: content/docs/0.12.x/operate/install/index.md [hunk 11]
```diff
@@ -403,6 +403,44 @@ Depending on whether you would like to install the execution plane for continuou
     </p>
     </details>
 
+    <details><summary>**Istio 1.8+ on OpenShift 3.11**</summary>
+    <p>
+
+    * Istio provides an Ingres Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
```
Comment (mowies) [inline content/docs/0.12.x/operate/install/index.md:409]:
```suggestion
    * Istio provides an Ingress Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
```

File: content/docs/0.11.x/operate/install/index.md [hunk 12]
```diff
@@ -422,6 +460,48 @@ Depending on whether you would like to install the execution plane for continuou
...
+
+    * To install an NGINX Ingress Controller, please refer to the [official documentation](https://kubernetes.github.io/ingress-nginx/).
+
+    * [Determine the ingress IP](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
+
+      ```console
+    kubectl -n ingress-nginx get svc ingress-nginx
+      ```
+
+    * Create an `ingress-manifest.yaml` manifest for an ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example of an `ingress-manifest.yaml` manifest shown next, `nip.io` is used as wildcard DNS for the IP address.)
```
Comment (mowies) [inline content/docs/0.11.x/operate/install/index.md:500]:
```suggestion
    * Create an `ingress-manifest.yaml` manifest for an ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example below, `nip.io` is used as wildcard DNS for the IP address.)
```

File: content/docs/0.11.x/operate/install/index.md [hunk 13]
```diff
@@ -406,6 +406,44 @@ Depending on whether you would like to install the execution plane for continuou
...
+
+    * Istio provides an Ingres Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
+
+    * [Determine the ingress IP](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
+
+      ```console
+    kubectl -n istio-system get svc istio-ingressgateway
+      ```
+
+    * Create an `ingress-manifest.yaml` manifest for an Ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example of an `ingress-manifest.yaml` manifest shown below, `nip.io` is used as wildcard DNS for the IP address.)
```
Comment (mowies) [inline content/docs/0.11.x/operate/install/index.md:420]:
```suggestion
    * Create an `ingress-manifest.yaml` manifest for an Ingress object in which you set IP-ADDRESS or your hostname and then apply the manifest. (**Note:** In the example below, `nip.io` is used as wildcard DNS for the IP address.)
```

File: content/docs/0.11.x/operate/install/index.md [hunk 14]
```diff
@@ -406,6 +406,44 @@ Depending on whether you would like to install the execution plane for continuou
     </p>
     </details>
 
+    <details><summary>**Istio 1.8+ on OpenShift 3.11**</summary>
+    <p>
+
+    * Istio provides an Ingres Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
```
Comment (mowies) [inline content/docs/0.11.x/operate/install/index.md:412]:
```suggestion
    * Istio provides an Ingress Controller. To install Istio, please refer to the [official documentation](https://istio.io/latest/docs/setup/install/).
```

**Review by mowies — CHANGES_REQUESTED** [review]

**Comment by odubajDT** [comment]
@mowies thank you, fixed :)

**Review by mowies — APPROVED** [review]

**Pull request merged**